backend/test/user.test.ts

```diff
@@ -93,3 +93,35 @@ describe('/userLoginSignup tests', () => {
     expect(updateUser).not.toHaveBeenCalled();
   });
 });
+
+describe('/updateUser tests', () => {
+  beforeEach(() => jest.clearAllMocks());
+  it('invalid faculty should return error', async () => {
```

Comment:
This covers the case where the faculty is empty but we also need a test for when the faculty is a value but not a valid value. E.g. if the value is `invalidFaculty`